### ISSUE TITLE
standalone image

### ISSUE THREAD

**Issue by afrozl** [oracle/fastr#130]
I've ex[experimented with the R native image and it appears to address the slow startup issues with fastR. The warm-up time is reduced by 50% and is low enough to justify using fastR for scripts that normally take longer than the overhead of the startup.

However, my goal is to deploy the R native image as a custom runtime for AWS lambda. Given the maximum size constraints inherent to lambda, I was wondering if there is a way to create a standalone binary for fastR?

I apologize if this isn't the appropriate forum to ask these types of questions.

**Comment by steve-s** [comment]
Hello,

thank you for reaching to us. If I understand it correctly, you are looking for something like [TruffleRuby standalone distribution](https://github.com/oracle/truffleruby/blob/master/doc/user/standalone-distribution.md)? Note that the size of compressed TruffleRuby standalone is around ~140MB -- would that be small enough?

**Comment by afrozl** [comment]
Yes, that would work, The AWS limit on runtime binary +  associated code is 250MB (uncompressed)
